sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/predicates.scala

```diff
@@ -518,7 +563,12 @@ case class GreaterThan(left: Expression, right: Expression)
 }
 
 @ExpressionDescription(
-  usage = "a _FUNC_ b - Returns TRUE if a is not smaller than b.")
+  usage = "expr1 _FUNC_ expr2 - Returns TRUE if expr1 is not smaller than expr2.",
```

Comment:
"not smaller" seems a little indirect, compared to describing this as "greater than or equal to"